README.md

```diff
@@ -165,6 +165,23 @@ class BlockchainInfo extends Driver {
 }
 ```
 
+### Instance methods
+#### `blockStatus(blockHeight)`
+Returns the promise with block status, message and proof(if exist), 
+could return an error in case of unexpected troubles.
```

Comment:
"Returns a promise with the block status, message and proof (if exists), can return an error in case of unexpected troubles."